eng/pipelines/coreclr/exploratory.yml

```diff
@@ -0,0 +1,63 @@
+# This job definition automates the exploratory tool.
+
+trigger:
+  batch: false
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - src/coreclr/jit/*
+    - src/coreclr/inc/jiteeversionguid.h
+
+# This pipeline is supposed to be run only on merged changes
+# and should not be triggerable from a PR. 
+pr: none
+
+schedules:
+- cron: "0 17 * * 0"
+  displayName: Sun at 9:00 AM (UTC-8:00)
+  branches:
+    include:
+    - main
```

Comment:
can we filter by paths on schedules? (I don't know if it is possible).

Reply:
Just changed it to nightly so don't need to filter by paths I suppose?